nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -158,36 +180,62 @@ class TfOpIdxMap {
     buffer_map.addDataWhenNotFound(
       empty_buffer, {0, empty_buffer}); /// this represents empty buffer
 
-    auto &variable_map = getIndexMap<const float *, const Tensor *>();
-    auto update_variables = [&variable_map,
-                             &buffer_map](const TfOpNode::Variables &variables,
-                                          bool update_buffer) {
-      for (auto &variable : variables) {
-        const float *buf = variable->getData();
-        variable_map.addDataWhenNotFound(buf, variable);
-        if (update_buffer) {
+    auto update_buffer_map =
+      [&buffer_map](const TfOpNode::Variables &variables) {
+        for (auto &variable : variables) {
+          const float *buf = variable->getData();
+          if (variable->getName().find("nntrainer_internal_perm") !=
```

Comment:
I wonder do we need this if state?

Reply:
No, actually, I wanted to update only Transpose op's buffer. I'll fix it soon.